src/Fixer/Operator/TernaryOperatorSpacesFixer.php

```diff
@@ -118,6 +122,32 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
         }
     }
 
+    /**
+     * @param int $index
+     *
+     * @return bool
+     */
+    private function belongsToAlternativeSyntax(Tokens $tokens, $index)
```

Comment:
static?

Reply:
nah, `belongsToGoToLabel` is not static either